src/integrations/skadi/api/v2/clients.ts

```diff
@@ -131,20 +131,25 @@ export class SkadiApiClientV2 implements ISkadiApiClientV2 {
     userId,
   }: CancelCampaignArgs): Promise<{ budget: string }> {
     return this.garmr.execute(async () => {
-      const { budget, error } = await fetchParse<CancelPostCampaignResponse>(
```

Comment:
The `fetchParse` function does not really check `response.ok`, so error 400 and such are not handled properly.